stream_alert/rule_processor/handler.py

```diff
@@ -57,9 +57,12 @@ def __init__(self, context):
         self._processed_size = 0
         self._alerts = []
 
+        rule_import_paths = [item for location in {'rule_locations', 'matcher_locations'}
+                             for item in self.config['global']['general'][location]]
+
         # Create an instance of the StreamRules class that gets cached in the
         # StreamAlert class as an instance property
-        self._rules_engine = RulesEngine(self.config)
+        self._rules_engine = RulesEngine(self.config, *rule_import_paths)
```

Comment:
Is there any advantage to expanding the paths as arguments instead of leaving them as a list? I have no strong preference either way, but I'm not sure why we do this here and not elsewhere

Reply:
we don't leverage arbitrary len arguments very often (if at all?) that I'm aware of which might be why it looks odd :)